decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
Hey @deivid-rodriguez . We tried `to_str` instead of `to_s` and it works like a charm. Can you change it please? 😄